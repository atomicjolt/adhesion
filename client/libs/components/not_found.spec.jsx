import React from 'react';
import { shallow } from 'enzyme';
import NotFound from './not_found';

describe('not found', () => {
  it('matches the snapshot', () => {
    const result = shallow(<NotFound />);
    expect(result).toMatchSnapshot();
  });
});
