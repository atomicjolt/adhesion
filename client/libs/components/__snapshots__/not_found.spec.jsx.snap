// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`not found matches the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <_DefaultExportValue />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h1>
          Page Not Found
</h1>,
        <p>
          Sorry, but the page you were trying to view does not exist.
</p>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Page Not Found",
        },
        "ref": null,
        "rendered": "Page Not Found",
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Sorry, but the page you were trying to view does not exist.",
        },
        "ref": null,
        "rendered": "Sorry, but the page you were trying to view does not exist.",
        "type": "p",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1>
            Page Not Found
</h1>,
          <p>
            Sorry, but the page you were trying to view does not exist.
</p>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Page Not Found",
          },
          "ref": null,
          "rendered": "Page Not Found",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Sorry, but the page you were trying to view does not exist.",
          },
          "ref": null,
          "rendered": "Sorry, but the page you were trying to view does not exist.",
          "type": "p",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactFifteenAdapter {
      "options": Object {
        "supportPrevContextArgumentOfComponentDidUpdate": true,
      },
    },
  },
}
`;
